single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -422,6 +425,8 @@ protected void drawBuses(String prefixId,
                         true, null));
             }
 
+            insertBusInfo(prefixId, graph.getCoord(), layoutParameters.getBusInfoMargin(), root, busNode, metadata, initProvider, styleProvider);
```

Comment:
I think it makes more sense to put the BusInfo directly inside the BusNode Element `g`. Just change `root` by `g` in this line, and move it above `root.appendChild(g)`